src/css/Contact.css

```diff
@@ -33,8 +48,132 @@
    
 } 
 
-.Contact-h4{
-        font-size: 2rem;
-        font-weight: 500;
-        margin-top: 0;
+
+/*The paragraph above the contact form*/
+.Contact-p2{
+    color: rgba(255,255,255,.9);
+    font-size: 1.2rem;
+    font-weight: 300;
+    margin-bottom: 2rem;
+}
+
+.icon-Text{
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    margin-bottom: 2rem;
+}
+
+.social-media{
+    display: flex;
+    width: 50%;
+    justify-content: space-between;
+}
+
+.icon-circle{
+    flex: 0 0 40px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+    border-radius: 50%;
+    transition: .3s all ease-in-out;
+}
+
+/*trying to have a hovering effect over the icon circles*/
+/*having an issue hovering over anything changes the color rather than the icon*/
+/* :hover{
+    background: #05f7ff;
+} */
+
+/*it should be calling icon-Text and then the fa class*/
+.fa{
+    flex: 0 0 20px;
+    font-size: 1.8rem;
+    margin-right: 2rem;
+    color: #05f7ff;
+}
+
+/*No class declared*/
+.Contact-span{
+    font-size: 1.3rem;
+    font-weight: 300;
+}
+
+/****Acutal form****/
+.Contact-form {
+    padding: 40px;
+    width: 100%;
+}
+
+.column{
+    display: flex;
+    width: 100%;
+    flex: 0 0 100%;
+}
+
+.form-group{
+    display: flex;
+    justify-content: flex-start;
+    flex-direction: column;
+    flex: 0 1 100%;
+    padding-right: 40px;
+} :nth-child(){ /**matches every element that is the nth child of its parent**/
+    padding-right: 0;
+}
+
+#radio-buttons{
+    display: flex;
+    justify-content: flex-start;
+    margin-top: 1rem;
+    margin-bottom: 2rem;
+} .solo{
+    padding: 0;
+}
+
+.Contact-rb/*input[type="radio"]*/:checked + label{
+    color: white;
+    font-weight: 700;
+}
+
+button{
```

Comment:
PR looks good, this just needs changed to be a classname, and your branch will need to get up to date with what is currently on main. I have time and don't want to make you go out of your way to do all that, so I'll go ahead and do it right now
<img width="1012" alt="Screen Shot 2022-09-21 at 9 25 38 AM" src="https://user-images.githubusercontent.com/46248021/191515925-1e093482-da8d-429b-8aae-2afddbb4d86b.png">